Sources/SwiftFormat/FormatPipeline.swift

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2018 Apple Inc. and the Swift project authors
```

Comment:
It looks like some files have been updated to 2019, but most have been untouched. Did you want to update them all at once or incrementally?